apps/demos/Demos/Menu/Scrolling/Angular/app/app.service.ts

```diff
@@ -2,1243 +2,345 @@ import { Injectable } from '@angular/core';
 
 export class Product {
   text: string;
-
   items?: Product[];
 }
 
```

Comment:
I don't see the failed tests for contextMenu demo. Could you please add tests for the demo, where contextMenu component is open?

Reply:
[3d154d2](https://github.com/DevExpress/DevExtreme/pull/27367/commits/3d154d27d4e44945263a101fa28db63207db580c)

[2b314e7](https://github.com/DevExpress/DevExtreme/pull/27367/commits/2b314e7a578bad32a37a410034304182d9994fd9)